src/Console/Ldap/SynchronizeUsersCommand.php

```diff
@@ -146,7 +154,7 @@ static function ($key, $value) {
 
         $restored_strategies = AuthLDAP::getLdapRestoredUserActionOptions();
         $description = sprintf(
-            __('Force strategy used for restored users (current configured action: "%s")'),
+            __('Force strategy used for restored users (current configured action: "%s").'),
```

Comment:
This is not consistent with other options that have no ending `.` on their description.
```suggestion
            __('Force strategy used for restored users (current configured action: "%s")'),
```